packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -151,11 +151,10 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
           <ComboboxInputWrapper {...getComboboxProps({role: 'combobox'})}>
             <ComboboxInputSelect
               {...getToggleButtonProps({tabIndex: 0})}
-              {...getInputProps({disabled, required, ref})}
+              {...getInputProps({disabled, required, ref, ...props})}
```

Comment:
Might be nice to a comment in code to remind future maintainers why we're spreading props, in absence of a test.